donut/templates/layout.html

```diff
@@ -1,49 +1,40 @@
-<!DOCTYPE html>
-<html lang="en">
-    <head>
-        <title>Donut</title>
-        
-        <link rel="stylesheet" type="text/css" 
-              href="{{ url_for('static', filename='css/style.css') }}" />
-        {# Google Fonts #}
-        <link href='https://fonts.googleapis.com/css?family=EB+Garamond|Roboto' rel='stylesheet' type='text/css'>
+{% extends "bootstrap/base.html" %}
 
-        {# Bootstrap core CSS #}
-        <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet">
-        {# Bootstrap theme #}
+{% block html_attribs %} lang="en"{% endblock %}
+{% block styles %}
+    {{ super() }}
+    <link rel="stylesheet" type="text/css"
+            href="{{ url_for('static', filename='css/style.css') }}" />
+    {# Google Fonts #}
+    <link href='https://fonts.googleapis.com/css?family=EB+Garamond|Roboto' rel='stylesheet' type='text/css'>
+    {# Just for debugging purposes. Don't actually copy this line! #}
```

Comment:
I guess we don't need this copy pasta

Reply:
Yeah that was in there before so I wasn't going to change it

Reply:
feel free to remove it